datagateway_api/src/search_api/helpers.py

```diff
@@ -74,8 +84,76 @@ def create_error_message(e):
     return wrapper_error_handling
 
 
+def get_score(entities, query, group, limit):
+    """
+    Gets the score on the given entities based in the query parameter and limit
+    """
+
+    try:
+        myobj = {
+            "query": query,
+            "group": Config.config.search_api.scoring_group,
+            "limit": limit,
+            "itemIds": list(map(lambda entity: entity["pid"], entities)),
+        }
+        response = requests.post(
+            Config.config.search_api.scoring_server, json=myobj, timeout=5
```

Comment:
I like the use of `timeout=`, so requests won't get stuck if the scoring server isn't working

Reply:
Not sure to understand, there's a timeout no?
timeout=5  

or what do you mean?

Reply:
Ah, a typo from me but I was complimenting the use of timeouts here :)